cypress/integration/ResetPassword/index.spec.ts

```diff
@@ -0,0 +1,25 @@
+describe('Reset password page', () => {
+  describe('given the email', () => {
+    it('renders the success message', () => {
+      cy.interceptApi('/me', {
```

Comment:
Why do we need to intercept this?

Reply:
Because if the user is logged in and visits this page, the user will get redirected to the dashboard. So we need to mock this API request to return an unauthorized response.